frontend/common/src/components/MissingSkills/MissingSkills.stories.tsx

```diff
@@ -1,12 +1,14 @@
 import React from "react";
 import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { filterSkillsByCategory } from "@common/helpers/skillUtils";
+import { SkillCategory } from "@common/api/generated";
```

Comment:
I don't think we would use the `@common` alias from inside common.  I think this is what the linter doesn't like.

Reply:
Another reason to do #5072!